Makefile

```diff
@@ -68,7 +68,7 @@ oci-image-tool:
 
 schema-fs:
 	@echo "generating schema fs"
-	@cd schema && echo -e "$$(cat ../.header)\n\n$$(go generate)" > fs.go
+	@cd schema && echo "$$(cat ../.header)\n\n$$(go generate)" > fs.go
```

Comment:
On Fri, Jun 03, 2016 at 12:57:12PM -0700, Stephen Day wrote:
> >  schema-fs:
> >  	@echo "generating schema fs"
> > -	@cd schema && echo -e "$$(cat ../.header)\n\n$$(go generate)" > fs.go
> > +	@cd schema && echo "$$(cat ../.header)\n\n$$(go generate)" > fs.go
>
> This is not a drive by change. This make target is broken with `-e`
> and removing it did not change the output.

This target is broken with -e *for you*, and removing it breaks the
output *for me*.  The current form also presumably works for
@jonboulle (who submitted it in #78).  So rather than changing the
folks who it's broken for, I'd like to either leave it alone, or fix
it so it works for everyone.